spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/orm/jpa/HibernateProperties.java

```diff
@@ -56,6 +57,8 @@
 	 */
 	private Boolean useNewIdGeneratorMappings;
 
+	private String archiveScanner;
```

Comment:
That's not really what I mind for this. I think that it is safer to disable that behaviour if no property is set so we shouldn't expose a property for this at all.